news/2024/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "The Followpoint: D I O, the Mastermind Behind osu! Tournaments"
+date: 2024-09-30 17:00:00 +0000
+series: The Followpoint
+---
+
+Hey there, do you like osu! tournaments? Then this interview read might just be for you!
+
+![](/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/banner.jpg)
+
+Welcome back to The Followpoint where we are back in business with another interview for you to enjoy! This time it's with [D I O](https://osu.ppy.sh/users/3958619), you've probably heard of him because of his outstanding contributions to many tournaments around the osu! tournament scene, so much in fact that it would probably take forever to list how many tournaments he's contributed to!
```

Comment:
```suggestion
Welcome back to The Followpoint where we are back in business with another interview for you to enjoy! This time it's with [D I O](https://osu.ppy.sh/users/3958619). You've probably heard of him because of his outstanding contributions to many tournaments around the osu! tournament scene. So much in fact, that it would probably take forever to list how many tournaments he's contributed to!
```